### PR TITLE
Add Android Polygon and Full Text

### LINKED ISSUE #466: Missing Polygon docs (Android)
The android section is missing `Polygon` documentation. Related PRs:

parse-community/Parse-SDK-Android#696
parse-community/Parse-SDK-Android#684

Not sure about other platforms.

### PR DESCRIPTION
**Pull request by dplewis**
Closes: https://github.com/parse-community/docs/issues/466

I also fixed some aggregate rest docs.

### CODE REVIEW

**Comment by dplewis** [comment]
@montymxb Can you look this over?

File: _includes/android/geopoints.md [hunk 1]
```diff
@@ -16,6 +16,42 @@ This point is then stored in the object as a regular field.
 placeObject.put("location", point);
 ```
 
+To retrieved `ParseGeoPoint` from an object
```
Comment (montymxb) [inline _includes/android/geopoints.md]: typo:
> To retrieve

Comment (montymxb) [reply _includes/android/geopoints.md]: Could add some punctuation at the end of `object` as well.

File: _includes/android/geopoints.md [hunk 2]
```diff
@@ -16,6 +16,42 @@ This point is then stored in the object as a regular field.
...
+ParsePolygon polygon = new ParsePolygon(points);
+```
+
+This point is then stored in the object as a regular field.
+
+```java
+placeObject.put("bounds", polygon);
+```
+
+To retrieved `ParsePolygon` from an object
```
Comment (montymxb) [inline _includes/android/geopoints.md]: typo:
> To retrieve
Punctuation after `object` as well.

File: _includes/android/queries.md [hunk 3]
```diff
@@ -210,6 +210,35 @@ The above example will match any `BarbecueSauce` objects where the value in the
 
 Queries that have regular expression constraints are very expensive. Refer to the [Performance Guide](#regular-expressions) for more details.
 
+### Full Text Search
+
+You can use `whereFullText` for efficient search capabilities. Text indexes are automatically created for you. Your strings are turned into tokens for fast searching.
+
+* Note: Full Text Search can be resource intensive. Ensure the cost of using indexes is worth the benefit, see [storage requirements & performance costs of text indexes.](https://docs.mongodb.com/manual/core/index-text/#storage-requirements-and-performance-costs).
+
+* Parse Server 2.5.0+
```
Comment (montymxb) [inline _includes/android/queries.md]: Should state
> **Requires** Parse Server 2.5.0+

**Review by montymxb — CHANGES_REQUESTED** [review]
Just a few typos, stylistic suggestions. Rest looks good.

File: _includes/android/geopoints.md [hunk 4]
```diff
@@ -16,6 +16,42 @@ This point is then stored in the object as a regular field.
 placeObject.put("location", point);
 ```
 
+To retrieved `ParseGeoPoint` from an object.
```
Comment (montymxb) [inline _includes/android/geopoints.md]: Still need to correct **retrieved** to **retrieve**. To improve the wording could be
> To **retrieve a** `ParseGeoPoint` from an object.

File: _includes/android/geopoints.md [hunk 5]
```diff
@@ -16,6 +16,42 @@ This point is then stored in the object as a regular field.
...
+ParsePolygon polygon = new ParsePolygon(points);
+```
+
+This point is then stored in the object as a regular field.
+
+```java
+placeObject.put("bounds", polygon);
+```
+
+To retrieved `ParsePolygon` from an object.
```
Comment (montymxb) [inline _includes/android/geopoints.md]: same thing here.

**Comment by dplewis** [comment]
@montymxb Changes made

**Review by montymxb — CHANGES_REQUESTED** [review]
Still those 2 things, should be good after that.

**Review by montymxb — APPROVED** [review]
Awesome :octocat: , we're all good!

**Comment by dplewis** [comment]
Sweet!

**Pull request merged**